server/sources.go

```diff
@@ -198,7 +198,7 @@ func (s *Service) sourceVersion(ctx context.Context, src *chronograf.Source) str
 	if err == nil {
 		return retVal
 	}
-	s.Logger.WithField("error", err.Error()).Info("Failed to retrieve database version")
+	s.Logger.WithField("error", err.Error()).WithField("url", src.URL).Info("Failed to retrieve database version")
```

Comment:
Updated - this would have been enough to catch the issue we were having - the problem was we didn't know which source was actually failing.

Reply:
Sure, makes sense